test/quadrature.jl

```diff
@@ -0,0 +1,19 @@
+@testitem "Simpson degress of exactness" begin
+    import DFTK: simpson, simpson_nonuniform
+
```

Comment:
Nice, but can you also add the negative case (like that higher degree are no longer correct)